_data/strings.yml

```diff
@@ -90,8 +90,7 @@ en: # English
 
     When you are authorized to use the Chia® brand and respect the Chia® brand guidelines, you help us ensure the strength of the brand, while protecting the entire community from being misled, confused or mistreated.
 
-    Please read over the following guidelines carefully, and never hesitate to reach out with questions or permission requests at hello@chia.net.
-
+    Please read over the following guidelines carefully, and never hesitate to reach out with questions or permission requests at [hello@chia.net](mailto:hello@chia.net).
     Happy Farming!

```

Comment:
Hmm, somehow this removes the blank line between the email address line and the 'Happy Farming!'.

Reply:
Yes, the blank line is deleted.  Recovering now.